### PR TITLE
RNA-SeQC v2.x.x

### PR DESCRIPTION
**Pull request by stekaz**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Code review requested, please @bioconda/core

### CODE REVIEW

File: recipes/rna-seqc/meta.yaml [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+  url: '{{base}}/v{{version}}/rnaseqc.v{{version}}.linux.gz'   # [linux]
+  sha256: '{{sha256linux}}'                                    # [linux]
 
-build:
-  number: 2
-  noarch: generic
+  url: '{{base}}/v{{version}}/rnaseqc.v{{version}}.osx.gz'     # [osx]
+  sha256: '{{sha256osx}}'                                      # [osx]
 
-requirements:
```
Comment (bgruening) [inline recipes/rna-seqc/meta.yaml]: You need to define the requirements. If this is now a python only package you should have python as run-requirement and pip and python as host requirement.

Comment (stekaz) [reply recipes/rna-seqc/meta.yaml]: Thanks very much for the review @bgruening. These are just static executables/binaries so there are no host or run requirements. I am of course happy to add an empty requirements block if needed, but I think that is redundant.

Comment (bgruening) [reply recipes/rna-seqc/meta.yaml]: Is it possible to build it from source? We recommend building from source to stay compatible with various old architectures.

Comment (stekaz) [reply recipes/rna-seqc/meta.yaml]: Yes - there's a "full source" tarball that includes the SeqLib submodule/dependency and some test data, IIRC. I'm happy to give this a go tomorrow. Cheers.

Comment (stekaz) [reply recipes/rna-seqc/meta.yaml]: Sorry - I think I see the problem with the test. Will try and fix tonight/tomorrow and let you know when I'm done.

Comment (stekaz) [reply recipes/rna-seqc/meta.yaml]: Done; but a code review for this guy would be very much appreciated. There are also a couple of python scripts that come bundled alongside the source and these have quite a few dependencies. Should these also be included in the build?

**Comment by dpryan79** [comment]
@bioconda-bot update

File: recipes/rna-seqc/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/rna-seqc/meta.yaml]: The question is mostly how vital those are. A lot of tools come with random R/python helper scripts that are never used. We can focus on getting the primary package working and fix helper scripts as the need arises.

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**